core/settings.py

```diff
@@ -63,6 +63,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django_extensions",
+    'django_cron',
```

Comment:
Please add django_cron and its version in the requirements.txt file

Reply:
@jbm1412 Added django-cron to requirements.txt, resolved merge conflicts, and ensured unnecessary log files are ignored. Let me know if any further changes are needed.
![image](https://github.com/user-attachments/assets/369fa8a6-bb70-433a-95ca-3d382295f3f4)